src/appleseed/renderer/modeling/bssrdf/randomwalkbssrdf.cpp

```diff
@@ -495,7 +495,7 @@ namespace
             assert(abs(cosine) <= 1.0f);
             const Basis3f basis(normal);
             const Vector2f tangent = sample_circle_uniform(s);
-            const float sine = sqrt(max(1.0f - cosine * cosine, 0.0f));
+            const float sine = std::sqrt(max(1.0f - cosine * cosine, 0.0f));
```

Comment:
`std::max()`

Feel free to fix these ones (those not from `<cmath>`) in another PR.

Reply:
I think `max` can be included here, even if it - strictly speaking - would not be part of `cmath`. But since we got e.g. `fmax` in it, I think it's fine to include `max`/`min` here as well.